Content.Server/Administration/Commands/SwapMindCommand.cs

```diff
@@ -0,0 +1,99 @@
+using Content.Shared.Administration;
+using Content.Shared.Mind;
+using Content.Shared.Mind.Components;
+using Robust.Server.Player;
+using Robust.Shared.Console;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Content.Server.Administration.Commands
+{
+    [AdminCommand(AdminFlags.Admin)]
+    sealed class SwapMindCommand : IConsoleCommand
```

Comment:
```suggestion
    sealed class SwapMindCommand : LocalizedCommands
```
this implements default strings for the description and help